api/src/services/BaseService.ts

```diff
@@ -22,5 +26,9 @@ export abstract class BaseService<TEntity extends Model>
 
     return data;
   }
+
+  public async findAllPaginated(page: number, pageSize: number) {
+      return await this.paginator.paginate(page, pageSize)
```

Comment:
W (Pedantic): Missing semicolon. Should be being picked up by pipeline. Issue to be created.

Reply:
Commit: 02dbeb7a0fa4367798ed906384e2bba6522c1763